caddyhttp/fastcgi/fastcgi_test.go

```diff
@@ -293,4 +296,24 @@ func TestBuildEnv(t *testing.T) {
 	envExpected["CUSTOM_URI"] = "custom_uri/fgci_test.php?test=blabla"
 	envExpected["CUSTOM_QUERY"] = "custom=true&test=blabla"
 	testBuildEnv(r, rule, fpath, envExpected)
+
+	// 6. Test Caddy-Rewrite-Original-URI header is not removed
+	const internalRewriteFieldName = "Caddy-Rewrite-Original-URI"
```

Comment:
Instead of redeclaring this here, let's move the const declaration from fastcgi.go out of a function so that this test file can use it too.